spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -552,6 +552,12 @@ def main(argv: Sequence[str]):
                          f" and segmentation [{img_seg.data.shape}]). "
                          f"Please verify that your compression labels and vertebral labels were done in the same space as your input segmentation.")
     path_ref = os.path.join(__data_dir__, 'PAM50_normalized_metrics')
+    # Fetch the subfolder that contains the "sub-{site}_{contrast}_PAM50.csv" files
+    path_ref_hc = next((folder for (folder, _, filenames) in os.walk(path_ref)
+                        if any((f.startswith('sub-') and f.endswith('.csv')) for f in filenames)), None)
+    if arguments.normalize_hc and not path_ref_hc:
+        raise FileNotFoundError(f"Directory with normalized PAM50 metrics {path_ref} does not contain any CSV files.\n"
+                                f"You can try re-downloading it using 'sct_download_data -d PAM50_normalized_metrics'.")
     # Check if path_ref with normalized metrics exists
     if arguments.normalize_hc and not os.path.isdir(path_ref):
         raise FileNotFoundError(f"Directory with normalized PAM50 metrics {path_ref} does not exist.\n"
```

Comment:
Sorry to nitpick, but I think this new check should come after the existing check. In other words:

- Check to see if the folder exists
- If it doesn't exist, we'll fail early.
- If it does, then we check if it contains CSV files.

I think right now, because of the order of the checks, if the directory doesn't exist, `os.walk` will return `[]`, and so the wrong error message will get printed ("CSV" error instead of "not exist" error).